Dockerfile

```diff
@@ -6,6 +6,8 @@ LABEL maintainer="Adam Schwalm <adam.schwalm@starlab.io>"
 RUN yum update -y && yum install -y \
     # Add the dnf plugins so we can enable PowerTools \
     dnf-plugins-core \
+    # Needed for cpuid and systemd-networkd \
```

Comment:
Can you clarify that this is needed for the installroot, not for things we actually install in the container itself

Reply:
```
Curl error (37): Couldn't read a file:// file for file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-8 [Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-8]
```

We need it in the container, as yum installroot epel-release followed by yum installroot systemd-networkd fails.
Yum looks for that rpm signing key in the build container even though we're using installroot

Reply:
Ah, sorry I was asking to updated the comment to explain that we don't need cpuid and systemd-networkd in the container, but rather for building stuff with it.